docs/users/installation.md

```diff
@@ -0,0 +1,225 @@
+- The `semanticdb-scalac` compiler plugin produces
+  [SemanticDB](https://github.com/scalameta/scalameta/blob/master/semanticdb/semanticdb3/semanticdb3.md)
+  files in the target directory `META-INF/semanticdb/`. These files are used by
+  Scalafix to understand symbols and types in Scala source code.
+- Beware that the `semanticdb-scalac` compiler plugin adds around 7-25% overhead
+  to compilation. It's recommended to provide generous JVM memory and stack
+  settings in `.jvmopts`:
+
+      -Xss8m
+      -Xms1G
+      -Xmx4G
+
+### Verify installation
+
+To verify the installation, check that scalacOptions and libraryDependencies
+contain the values below.
+
+```scala
+> show scalacOptions
+[info] * -Yrangepos
+> show libraryDependencies
+[info] * org.scalameta:semanticdb-scalac:@SCALA212@:plugin->default(compile)
+```
+
+### Example project
+
+For a minimal example project using sbt-scalafix, see the
+[scalacenter/scalafix-sbt-example](https://github.com/scalacenter/scalafix-sbt-example)
+repository.
+
+```scala
+git clone https://github.com/olafurpg/scalafix-sbt-example
+cd scalafix-sbt-example
+sbt "scalafix RemoveUnusedImports"
+git diff // should produce a diff
+```
+
+### Settings and tasks
+
+| Name                    | Type           | Description                                                                                                                               |
+| ----------------------- | -------------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
+| `scalafix <rule>..`     | `Unit`         | Run scalafix on project sources. See [Rules](rules/overview.md) or use tab completion to explore supported rules.                         |
+| `scalafixCli <args>`    | `Unit`         | Invoke scalafix command line interface directly. See {% doc_ref Installation, cli %} or use tab completion to explore supported arguments |
+| `scalafixTest <rule>..` | `Unit`         | Similar to `scalafix` except does not write to files, only reports an error if the task would produce a diff.                             |
+| `sbtfix <rule>..`       | `Unit`         | Run scalafix on the `*.sbt` and `project/*.scala` sources of this build. Only supports syntactic rules.                                   |
+| `sbtfixTest <rule>..`   | `Unit`         | Similar to `sbtfix` except does not write to files, only reports an error if the task would produce a diff.                               |
+| `scalafixConfig`        | `Option[File]` | .scalafix.conf file to specify which scalafix rules should run. Defaults to `.scalafix.conf` in the root directory if it exists.          |
+
+### FAQ
+
+- How to exclude/add files that are processed by the `scalafix` task?
+
+```
+unmanagedSources.in(Compile, scalafix) := unmanagedSources.in(Compile).value.filter(file => ...)
+```
+
+## Command line
+
+The recommended way to install the scalafix command-line interface is with
```

Comment:
I would remove recommended, since it's the only sane way to install the cli.

Reply:
Good catch, previously there were several alternatives to install the cli (homebrew, wget). Coursier only is simpler.